src/index.html

```diff
@@ -2,21 +2,70 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous" />
     <link
-      rel="stylesheet"
-      href="./style.css"
-    />
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet" />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <a href="./index.html" class="header_link_logo">
+        <img src="./images/logo.png" alt="logo moyo brand" />
+      </a>
+      <nav class="header-nav">
+        <ul class="nav-list">
+          <li class="nav-list_item">
```

Comment:
Remember to add empty lines between sibling elements that span multiple lines. This improves readability. In this case, you should add empty lines between the `li` elements.